css/webviews.css

```diff
@@ -32,3 +32,40 @@ body.windows #webviews {
 	height: 100vh;
 	z-index: 99;
 }
+
+.arrow-indicator {
+	width: 140px;
+	height: 140px;
+	border-radius: 70px;
+	background: rgba(0,0,0,0.8);
+	position: fixed;
+	top: 50%;
+	z-index: 1;
+	transition: opacity 0.3s, visibility 0.3s;
```

Comment:
There's no transition for ```visibility```, so I think that part can be removed without affecting anything.